frontend/src/components/Chat.js

```diff
@@ -26,6 +25,10 @@ const Chat = ({ room }) => {
 
   const scrollPositionRef = useRef(null)
 
+  useEffect(() => {
+    collabSocket.emit('join-room', room, collabSocket.id)
+  }, [room])
+
```

Comment:
I don't think we need to join the room here cos Chat is a child of RenderPage and we already joined the room when that component is mounted?

Reply:
You are correct! tested it as well